waku/node/v2/waku_types.nim

```diff
@@ -11,10 +11,13 @@ import
   libp2p/switch,
   libp2p/stream/connection,
   libp2p/protocols/pubsub/[pubsub, gossipsub],
-  nimcrypto/sha2
+  nimcrypto/sha2,
+  sqlite3_abi
+
 # constants required for pagination -------------------------------------------
-const MaxPageSize* = 100 # Maximum number of waku messages in each page 
+const MaxPageSize* = uint64(100) # Maximum number of waku messages in each page 
```

Comment:
What is this about? Seems like this belongs to whatever proc needs it to be uint64